docker-compose.yml

```diff
@@ -1,69 +1,74 @@
-worker: &worker
-  image: addons/addons-server
-  command: supervisord -n -c /code/docker/supervisor-celery.conf
-  entrypoint: ./scripts/start-docker.sh
-  volumes:
-    - .:/code
-  links:
-    - memcached
-    - mysqld
-    - elasticsearch
-    - redis
-    - rabbitmq
-  environment:
-    - BROKER_URL=amqp://olympia:olympia@rabbitmq/olympia
-    - CELERY_RESULT_BACKEND=redis://redis:6379/1
-    - DATABASE_URL=mysql://root:@mysqld/olympia
-    - ELASTICSEARCH_LOCATION=elasticsearch:9200
-    - MEMCACHE_LOCATION=memcached:11211
-    - MYSQL_DATABASE=olympia
-    - MYSQL_ROOT_PASSWORD=docker
-    - OLYMPIA_SITE_URL=http://olympia.dev
-    - PYTHONDONTWRITEBYTECODE=1
-    - PYTHONUNBUFFERED=1
-    - RECURSION_LIMIT=10000
-    - REDIS_LOCATION=redis://redis:6379/0?socket_timeout=0.5
-    - TERM=xterm-256color
+version: "2"
 
-web:
-  <<: *worker
-  command: supervisord -n -c /code/docker/supervisor.conf
+services:
+  worker: &worker
+    image: addons/addons-server
+    command: supervisord -n -c /code/docker/supervisor-celery.conf
+    entrypoint: ./scripts/start-docker.sh
```

Comment:
Does that work now? 😨

Reply:
Yes! Remember it was the fact that we couldn't mount within a docker image.

Reply:
Ha, I do! Completely forgot about that 😆